docs/_reporting/power-bi/README.md

```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
+| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plan recommendations               |                ❌                 |                🔜                 |             ✅                     |              🔜               |
+| Supports multiple scopes                            |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports scopes in different tenants<sup>5</sup>    |          ⚠️<sup>5</sup>           |          ⚠️<sup>5</sup>           |             ❌                     |          ⚠️<sup>5</sup>(via Hubs) |
+| Faster data load times                              |                🔜                 |                ✅                 |             ✅                     |         ✅ (via Hubs)         |
+| Supports >$65M in cost details                      |                ❌                 |                ✅                 |             ❌                     |              ✅               |
+| Accessible outside of Power BI                      |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Kusto Query Language (KQL) support                        |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Native integration with cost optimization workbook |         ❌                 |                ❌                 |             ✅<sup>6</sup>                     |              ❌               |
+| Azure Data Explorer built-in dashboards                    |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Learn more                                          |      [Learn more][about-rawexports]    |     [Learn more][about-hubs]     | [Learn more][about-hubs]          | [Learn more][about-hubs]       |
 
 [about-connector]: https://aka.ms/costmgmt/powerbi
 [about-hubs]: ../hubs/README.md
+[about-rawexports]: ../power-bi/setup.md
 [hubs-template]: ../hubs/template.md
 
 _<sup>1) Microsoft Fabric can connect to either raw exports or FinOps hubs. FinOps toolkit reports do not support Microsoft Fabric yet but will in a future release.</sup>_
 
-_<sup>2) Power BI constraints are based on data size and processing time. Monitored spend estimations are for reference only. You may see different limits based on services you use and other datasets you ingest.</sup>_
+<sup>2) The estimated cost of Hubs on ADX assumes the two instances are running 24x7. Costs can be reduced by enabling auto-shutdown outside of business hours or applying commitment discounts. Prices are estiamted in UK South as of October 2024</sup>_
```

Comment:
Can you reword this to say that the estimate assumes a single node dev/test cluster and that they may need to consider additional nodes for larger accounts? I don't know what the precise guidance should be, so we'll have to be hand-wavy.